serverless.yml

```diff
@@ -21,13 +21,36 @@ plugins:
   - serverless-webpack
   - serverless-offline-scheduler
   - serverless-offline
+  - serverless-plugin-aws-alerts
 
 custom:
   serverless-offline:
     port: 4000
   webpack:
     webpackConfig: ./webpack.config.js
     includeModules: true
+  notifications:
+    - protocol email
+      endpoint freecodecampalerts+otto.jongerius@gmail.com
```

Comment:
Just a note, to move this over to variables.

Reply:
How do we have multiple endpoints?

For example, how to we notify the entire developer team via individual email plus the team account?

Is there a scope for integrating with the twillio api for critical alerts? i.e. the entire api is not responding.

Reply:
@raisedadead will do.

@Bouncey are you asking how this would be possible? If so: a notification is sent to an SNS (AWS Simple Notification Service) topic. Basically pub/sub messaging. A topic can have one or more subscribers. Lots of destinations are possible, including sms, but phone calls are AFAK not supported. However, as you hinted at, we can hook it up to Twillio. Do we have a Twillio account at the moment? I'd be happy to set that up.